doc/api/perf_hooks.md

```diff
@@ -1722,6 +1616,78 @@ added:
 
 Returns a {RecordableHistogram}.
 
+## `perf_hooks.eventLoopUtilization([utilization1[, utilization2]])`
+
+<!-- YAML
+added: REPLACEME
```

Comment:
I think the old ones can use a changes entry about their new home (I suppose deprecation would follow?)